### PR TITLE
Prep for 3.10 beta release: Use the `main` branch

### PR DESCRIPTION
**Pull request by Mariatta**
Closes https://github.com/python/miss-islington/issues/446

Signed-off-by: Mariatta <mariatta@python.org>

### CODE REVIEW

**Comment by Mariatta** [comment]
@pablogsal This should allow it to work with the `main` branch. Please merge whenever we start using the `main` branch.

File: miss_islington/tasks.py [hunk 1]
```diff
@@ -21,6 +21,14 @@
 
 cache = cachetools.LRUCache(maxsize=500)
 
+CHERRY_PICKER_CONFIG = {
+    "team": "python",
+    "repo": "cpython",
+    "check_sha": "7f777ed95a19224294949e1b4ce56bbffcb1fe9f",
```
Comment (pablogsal) [inline miss_islington/tasks.py:27]: One small question: what is this SHA for?

Comment (Mariatta) [reply miss_islington/tasks.py:27]: https://github.com/python/cherry-picker/blob/39e8d6d10eecae6b6d096e3e03b69c5104ba5507/cherry_picker/cherry_picker.py#L490-L493

We needed a way to detect whether cherry-picker is working on the right git repo. We use this to detect that we are indeed in CPython repo.  If not mistaken, I think this SHA is the first commit to CPython.

Comment (pablogsal) [reply miss_islington/tasks.py:27]: Thanks for the clarification!

**Comment by pablogsal** [comment]
:rocket:  Thanks @Mariatta :heart:

**Pull request merged**